spec/javascripts/stimulus/enterprise_form_controller_test.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import { Application } from "stimulus";
+import side_menu_controller from "../../../app/webpacker/controllers/side_menu_controller";
```

Comment:
I'm wondering if there is any way to test `enterprise_form_controller` controller without `side_menu_controller` ?

 - If yes, I think it should be done
 - If no, I think `enterprise_form_controller` should be included to `side_menu_controller`.

What. do you think?